test/e2e/framework/volume_util.go

```diff
@@ -350,8 +350,9 @@ func VolumeTestCleanup(f *Framework, config VolumeTestConfig) {
 	}
 
 	if config.ServerImage != "" {
-		if err := f.WaitForPodTerminated(config.Prefix+"-client", ""); !apierrs.IsNotFound(err) {
-			ExpectNoError(err, "Failed to wait client pod terminated: %v", err)
+		err := f.WaitForPodNotFound(config.Prefix+"-client", PodDeleteTimeout)
+		if err != nil {
+			ExpectNoError(err, "Failed to wait client pod not found: %v", err)
```

Comment:
a better error message could be "Failed to wait for client pod deletion:"

same for the others

Reply:
Fixed